### DISCUSSION TITLE
能否加一个:注册皮肤站时需要验证邮箱

### DISCUSSION THREAD

**Discussion by Yukino12333** [bs-community/blessing-skin-server#448] [Ideas]
现在似乎连验证邮箱的可用性都做不到，注册一堆空号，难以管理

**Comment by Big-Cake-jpg** [comment]
站点配置
![image](https://user-images.githubusercontent.com/76810494/180627619-58a61a4d-64ce-493c-9bfa-13e1078ff83b.png)
